roles/patroni/tasks/custom_wal_dir.yml

```diff
@@ -43,8 +43,13 @@
         dest: "{{ postgresql_wal_dir }}/"
       delegate_to: "{{ inventory_hostname }}"
 
-    - name: "Rename {{ 'pg_wal' if postgresql_version is version('10', '>=') else 'pg_xlog' }} to {{ 'pg_wal' if postgresql_version is version('10', '>=') else 'pg_xlog' }}_old"
-      ansible.builtin.command: mv {{ postgresql_data_dir }}/{{ 'pg_wal' if postgresql_version is version('10', '>=') else 'pg_xlog' }} {{ postgresql_data_dir }}/{{ 'pg_wal' if postgresql_version is version('10', '>=') else 'pg_xlog' }}_old
+    # 🔄 Determine base pg_wal_dir name
+    - name: Roles.patroni.custom_wal_dir | Set pg_wal_dir based on postgresql_version
```

Comment:
@vitabaks

Similarly, I follow the same principle for naming tasks (although I avoid using emojis in the actual code, only in comments or commits). I strive to make the code more readable and easily traceable to its corresponding task

This approach simply aligns with the file hierarchy

Please refer to the image for clarification

![image](https://github.com/vitabaks/postgresql_cluster/assets/93382375/9bd0fa77-9ff3-4587-95de-ab47de7106b1)